src/mongoose_scram.erl

```diff
@@ -51,14 +52,17 @@
 -define(SCRAM_SERIAL_PREFIX, "==SCRAM==,").
 -define(MULTI_SCRAM_SERIAL_PREFIX, "==MULTI_SCRAM==,").
 -define(SCRAM_SHA_PREFIX, "===SHA1===").
```

Comment:
It is possible to replace SCRAM_SHA_PREFIX to SCRAM_SHA1_PREFIX?

Reply:
Sure, it's possible to change but please note that this is a definition of a macro. This is only code internal format and doesn't put any implications on the configuration. We already were discussing the usage of sha and sha-1 in terms of consistency: 
[https://github.com/esl/MongooseIM/pull/2713#issuecomment-613980076](url)

Reply:
Yes you can see other ;)
- SCRAM_SHA224_PREFIX
- SCRAM_SHA256_PREFIX
- SCRAM_SHA384_PREFIX
- SCRAM_SHA512_PREFIX

I know that sha(1) = sha in Erlang crypto but...
- https://erlang.org/doc/man/crypto.html#type-sha1

Better to have:
- SCRAM_SHA1_PREFIX